packages/drivers/odsp-driver/src/odspDocumentServiceFactoryCore.ts

```diff
@@ -47,7 +47,7 @@ export class OdspDocumentServiceFactoryCore implements IDocumentServiceFactory {
     private readonly nonPersistentCache = new NonPersistentCache();
 
     public async createContainer(
-        createNewSummary: ISummaryTree,
+        createNewSummary: ISummaryTree | undefined,
```

Comment:
I do not see same change to IDocumentServiceFactory.createContainer. It should fail compilation, right (in loader)?